Automattic/sensei
# Importer: Associate courses with lessons from courses file

### Changes proposed in this Pull Request

* Moves lesson module assignment and course/lesson association to a new `Sensei_Import_Associations` post-processing task.
* Removes the Course column from lessons.
* Sets lessons and lesson order from the `Lessons` column in the courses file. Appropriately sets module lesson order when appropriate.

### Testing instructions

Testing note: I think there is a bug on the lesson order page that requires you drag things around before saving (even if it ends up being the same). If you load the lesson order page and don't change anything, sometimes that won't actually save the order you see by default.

* Use @gkaragia's export PR #3537 to create exported files. You will need to remove the `Course` column from the lesson export (that column should be removed once this PR is merged alongside that PR). Make sure you have a mix of courses with modules and without. Add multiple lessons to the modules and some without modules and reorder.
* Import those files and verify the lessons are correctly assigned to the courses and modules. Verify the order is maintained.
